pkg/apis/configuration/validation/validation.go

```diff
@@ -57,6 +57,33 @@ func validateTLS(tls *v1alpha1.TLS, fieldPath *field.Path) field.ErrorList {
 	return validateSecretName(tls.Secret, fieldPath.Child("secret"))
 }
 
+func validatePositiveInt(n *int, fieldPath *field.Path) field.ErrorList {
```

Comment:
positive integers do not include 0. that's why it's better to change the name as well. For example, `validatePositiveIntOrZero`